sdks/java/core/src/test/java/org/apache/beam/sdk/io/TextIOWriteTest.java

```diff
@@ -733,7 +733,7 @@ public void testWriteUnboundedWithCustomBatchParameters() throws Exception {
         LINES2_ARRAY,
         null,
         null,
-        3,
+        0,
         baseFilename,
```

Comment:
this test is exercises by different runner's vallidaterunner test suite. Only SamzaRunner fails. Would this change break other runners test?

Probably we just need to exclude this specific test from validateRunner target in runners/samza/build.gradle

Reply:
It doesn't break other tests (tested with DirectRunner). If we pass 0, then it searches for `file.txt*`.

Reply:
I see, then it will no longer assert if the GroupintoBatches works as intended for those runners currently passing this test. Still think exclude from SamzaRunner test suite is preferred for now

Reply:
Done